docs/UserGuide.md

```diff
@@ -210,6 +225,15 @@ Examples:
 * `sort n/` sorts the displayed list in ascending name.
 * `sort p/` sorts the displayed list in ascending phone.
 
+### Redoing an Undo : `redo`
+
+If the previous command was an `undo`, then will restore EmployeeManager to the state before `undo`.
+
+Format: `redo`
+
+Examples:
+* Adding `John`, `undo` and then `redo` will return `John` back into EmployeeManager.
```

Comment:
should we have something like "in that order will return `John` back into EmployeeManager", or something like "Adding `John`, followed by `undo` and then `redo`..."

Reply:
ok! done!